projects/kit/components/arrow/arrow.component.ts

```diff
@@ -28,4 +28,14 @@ export class TuiArrowComponent {
     }
 }
 
+export interface TuiSelectArrowMode {
+    interactive: PolymorpheusContent;
+    disabled: PolymorpheusContent;
+}
+
 export const TUI_ARROW = new PolymorpheusComponent(TuiArrowComponent);
+
+export const TUI_SELECT_ARROW_MODE: InjectionToken<TuiSelectArrowMode> =
```

Comment:
Let's drop `SELECT` from the name since it's relevant for other dropdowns too.

Reply:
fixed